docs/Home.md

```diff
@@ -1,58 +1,72 @@
-- [Block Shapes](Block-Shapes.md) - defining 3D meshes via definitions in JSON!
+## Getting Started as a Contributor
 
-## Announcement Channels
+🚀 Ready to dive into development? Check out the [Contributor Quick Start](Contributor-Quick-Start.md) guide to set up your workspace, understand the required tools, and make your first contribution.
 
-We have several ways to get the word out on updates, likewise, there are several ways to contact us.
+- Get familiar with [Git and GitHub basics](https://docs.github.com/en/get-started/quickstart/hello-world) for collaborating on code.
+- Set up your Terasology Development Workspace and start playing around with the code.
 
-- [Discord](https://discordapp.com/invite/terasology) - New development/game topics will be posted in `#announcement`, and any questions answered.
-- [GitHub (Engine)](https://github.com/MovingBlocks/Terasology) - "Watch" the official project here to be able to easily spot core commits and changes.
-- [GitHub (Modules)](https://github.com/Terasology) - "Watch" the module repos to be able to keep track of game content fixes / changes.
-- [Forum](http://forum.terasology.org/) - Find the progress reports of ongoing and past GSoC projects along with a lot of gameplay ideas and lore
-- [Twitter](http://twitter.com/#!/Terasology) - We'll tweet regularly about significant commits or new discussion topics posted, so "Follow" us for updates.
-- [Facebook](http://www.facebook.com/pages/Terasology/248329655219905) - If you prefer to keep updated via Facebook you can "Like" us on there to keep up.
-- [Jenkins RSS](http://jenkins.terasology.org/rssAll) - If you really want to know when something has just been built ;-)
+## Troubleshooting
+
+Encountering issues? Our [Troubleshooting guide](Troubleshooting.md) offers solutions for common problems faced by players and developers alike.
+
+- Find quick fixes and helpful resources for common issues.
+- Get additional support by reaching out on [Discord](http://discord.gg/Terasology) or our [Support Forum](http://forum.terasology.org/forum/support.20).
+
+## Stay Updated & Reach Out
+
+📢 We love sharing updates and hearing from you!
+
+- **[Discord](https://discordapp.com/invite/terasology)** - For `#announcements` and direct Q&A.
+- **[GitHub (Engine)](https://github.com/MovingBlocks/Terasology)** - Watch the engine or modules for real-time updates.
+- **[GitHub (Modules)](https://github.com/Terasology)** - Watch the module repos for game content fixes / changes.
+- **[Forum](http://forum.terasology.org/)** - Find the progress reports of ongoing and past GSoC projects
+- **[Twitter](http://twitter.com/#!/Terasology)** - Follow for tweet-sized news.
+- **[Facebook](http://www.facebook.com/pages/Terasology/248329655219905)** - Like us for updates in your feed.
+- **[Jenkins RSS](http://jenkins.terasology.org/rssAll)** - For the keen observers of new builds.
```

Comment:
`jenkins.terasology.org` has not been in use for a long time. `https://jenkins.terasology.io/rssAll` would be the current URL.

Reply:
noted